cEP-0015.md

```diff
@@ -0,0 +1,93 @@
+corobo Enhancement (security, tests and configurability)
+========================================================
+
+|Metadata|                                                           |
+|--------|-----------------------------------------------------------|
+|cEP     |15                                                         |
+|Version |1.0                                                        |
+|Title   |corobo Enhancement (security, tests and configurability)   |
+|Authors |Nitanshu Vashistha <nitanshu.vzard@gmail.com>              |
+|Status  |Proposed                                                   |
+|Type    |Process                                                    |
+
+Abstract
+--------
+
+This cEP describes the details of enhancement of [corobo](https://github.com/coala/corobo) in terms of security, tests, configurability and
+the new plugins that are to be added to corobo as part of the [GSoC Project](https://summerofcode.withgoogle.com/projects/#6603667076022272).
```

Comment:
I'd suggest to improve the project description on GSoC website as well, while you can.. maybe after the end of community bonding period.

Reply:
I didn't know we can update the description :o Thank's for informing, its done :+1: